src/components/audioManagement/audioManagement.tsx

```diff
@@ -130,6 +130,7 @@ class AudioManagement extends React.Component<AllProps> {
       prevProps.loopStart !== this.props.loopStart ||
       prevProps.loopEnd !== this.props.loopEnd
     ) {
+      this.getPlayer()?.setSeconds(this.props.loopStart);

```

Comment:
It would be better to simply use `this.getPlayer().seek(this.props.loopStart);` here.